.pre-commit-config.yaml

```diff
@@ -1,38 +0,0 @@
-default_language_version:
```

Comment:
why are you removing this one?

Reply:
A YAML file already exist in the backend folder. During the debugging of the linter issue, I moved it in the root folder. 
So, I removed it afterwards because I wanted to be sure it is okay to move it to the root folder permanently. Additionally, having it in the root folder streamlines linter checks during pre-commit.